src/Samples/SarifToCsv/SarifCsvColumnWriters.cs

```diff
@@ -71,6 +72,9 @@ public static Action<WriteContext> GetWriter(string columnName)
             writers["Tags"] = (c) => { c.Writer.Write(String.Join("; ", ((IEnumerable<string>)c.Result.Tags) ?? Array.Empty<string>())); };
             writers["WorkItemUris"] = (c) => { c.Writer.Write(String.Join("; ", c.Result.WorkItemUris?.Select((uri) => uri.ToString()) ?? Array.Empty<string>())); };
 
+            // Formatted Message
+            writers["FormattedMessage"] = (c) => { c.Writer.Write(c.Result.GetMessageText(c.Result.GetRule(c.Run))); };
```

Comment:
>FormattedMessage [](start = 21, length = 16)

Any reason not to just call this "Message"? That's more understandable to a human user who doesn't know that "formatted message" is a "thing" in the SARIF spec.

Reply:
I chose names to try to be consistent with the object model properties or methods, so you know how to get the thing you asked for from code.

I don't mind Message and will rename it.

Reply:
Your rationale makes sense (consistency with SARIF OM). Feel free to revert.

---
In reply to: [325837196](https://github.com/microsoft/sarif-sdk/pull/1672#discussion_r325837196) [](ancestors = 325837196)